public/index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Explore up-to-date genome data and monitor variants of SARS-CoV-2! covSPECTRUM is a fully interactive
+      content="Covid 19 variants and genome data explorer! covSPECTRUM is a fully interactive
```

Comment:
```suggestion
      content="SARS-CoV-2 variants and genome data explorer! covSPECTRUM is a fully interactive
```

We are not looking at Covid-19 (the disease) variants but at SARS-CoV-2 (the virus) ;)